sklearn/inspection/_permutation_importance.py

```diff
@@ -112,6 +132,10 @@ def permutation_importance(estimator, X, y, *, scoring=None, n_repeats=5,
         importances : ndarray, shape (n_features, n_repeats)
             Raw permutation importance scores.
 
+        If there are multiple scoring metrics in the scoring parameter
```

Comment:
Could you update the type in l.132 of importances in the first line:

```python
importances : ndarray of shape (n_features, n_repeats) or dict of such arrays
```

Reply:
I thought importances alone is not a dict of arrays in case of multiple scorers. As written at line 135, the result is a dict of Bunch.

Reply:
You a right I mess-up, we need to update results

```python
result : :class:`~sklearn.utils.Bunch` or dict of such instances
```

Reply:
On a side note, could you as well update the occurrences of `ndarray, shape`  to `ndarray of shape`

Reply:
done